datasources/src/test/scala/com/normation/plugins/datasources/UpdateHttpDatasetTest.scala

```diff
@@ -41,65 +41,42 @@ import ch.qos.logback.classic.Level
 import com.normation.BoxSpecMatcher
 import com.normation.box.*
 import com.normation.errors.*
-import com.normation.eventlog.EventActor
-import com.normation.eventlog.ModificationId
+import com.normation.eventlog.{EventActor, ModificationId}
 import com.normation.inventory.domain.NodeId
-import com.normation.plugins.AlwaysEnabledPluginStatus
-import com.normation.plugins.PluginEnableImpl
+import com.normation.plugins.{AlwaysEnabledPluginStatus, PluginEnableImpl}
 import com.normation.plugins.datasources.DataSourceSchedule.*
 import com.normation.rudder.domain.eventlog.*
-import com.normation.rudder.domain.policies.GlobalPolicyMode
-import com.normation.rudder.domain.policies.PolicyMode
-import com.normation.rudder.domain.policies.PolicyModeOverrides
-import com.normation.rudder.domain.properties.CompareProperties
-import com.normation.rudder.domain.properties.GenericProperty
+import com.normation.rudder.domain.policies.{GlobalPolicyMode, PolicyMode, PolicyModeOverrides}
 import com.normation.rudder.domain.properties.GenericProperty.*
-import com.normation.rudder.domain.properties.GlobalParameter
-import com.normation.rudder.domain.properties.NodeProperty
-import com.normation.rudder.facts.nodes.ChangeContext
-import com.normation.rudder.facts.nodes.CoreNodeFact
-import com.normation.rudder.facts.nodes.CoreNodeFactRepository
-import com.normation.rudder.facts.nodes.NodeFactChangeEvent
-import com.normation.rudder.facts.nodes.NodeFactChangeEventCallback
-import com.normation.rudder.facts.nodes.NodeFactChangeEventCC
-import com.normation.rudder.facts.nodes.NodeFactRepository
-import com.normation.rudder.facts.nodes.NodeSecurityContext
-import com.normation.rudder.facts.nodes.NoopFactStorage
-import com.normation.rudder.facts.nodes.NoopGetNodesBySoftwareName
-import com.normation.rudder.facts.nodes.QueryContext
+import com.normation.rudder.domain.properties.{GenericProperty, GlobalParameter, NodeProperty}
+import com.normation.rudder.facts.nodes.*
 import com.normation.rudder.repository.RoParameterRepository
 import com.normation.rudder.services.nodes.PropertyEngineServiceImpl
-import com.normation.rudder.services.policies.InterpolatedValueCompilerImpl
-import com.normation.rudder.services.policies.NodeConfigData
+import com.normation.rudder.services.policies.{InterpolatedValueCompilerImpl, NodeConfigData}
```

Comment:
can you change you intellij setting to keep import one per line, which is the config of our linter? You can replace by "*" when there is too many import, for ex above 3.